src/krux/psbt.py

```diff
@@ -275,9 +275,38 @@ def _classify_output(self, out_policy, out):
             return SELF_TRANSFER
         return SPEND
 
+    def _get_resume_fee(self, inp_amount, out_amount, output_policy_count):
```

Comment:
One detail Jeff used to ask me: Please don't add "get" in function names, to be more "pythonic"

Reply:
This is just an internal function, it even don't need comments because of the `_` prefix, so I don't think it needs to be more or less "pythonic", but... what do you suggest to be more "pythonic"?